lib/fauna/query.rb

```diff
@@ -8,14 +8,34 @@ module Fauna
   # Example:
   #
   #   query = Fauna::Query.create(Fauna::Ref.new('classes', 'spells'), Fauna::Query.quote(data: { name: 'Magic Missile' }))
+
+  ##
```

Comment:
You need to place this above the doc above, which is the doc for the `Fauna::Query` module.